### DISCUSSION TITLE
How do we globally redirect STDERR logs to STDOUT with Nuke v9.x ?

### DISCUSSION THREAD

**Discussion by SonnyRR** [nuke-build/nuke#1460] [Q&A]
Hey, as mentioned in the FAQ section, as well as in several issues in this repo, tools like `Docker CLI` & `Git` seems to write to `STDERR`. There was a way to globally override their loggers, so that the output is redirected to `STDOUT`. This seems to be obsolete now, since these logger properties are missing from the generated tool documents.

I've tried looking through the recent change logs, issues and discussions, but I cannot seem to understand why the `Action<OutputType, string>` delegates were removed from the `DataClassGenerator.cs`.

What would be the new way to redirect the output one-time if we have multiple targets that utilize `Docker CLI` or `Git` statements ? `ToolOptions.Extensions.SetProcessLogger<T>(Action<OutputType, string>)` is marked as obsolete.

**Comment by matkoch** [comment]
See https://github.com/nuke-build/nuke/discussions/1455#discussioncomment-11477104
